metricbeat/module/beat/docs.asciidoc

```diff
@@ -7,4 +7,4 @@ The default metricset is `stats`.
 [float]
 === Compatibility
 
-The Beat module is tested with Metricbeat 7.2 and is expected to work with all 7.x versions.
+The Beat module is expected to work with all versions of Beats >= 7.3.0.
```

Comment:
This comment applies to all the changes:

Is it necessary to say "expected to work"? If it doesn't work, it's a bug, right? 

Also I don't personally like the use of symbols over plain language (maybe because it requires extra parsing for my right brain).  I would avoid the weasel words. How about something simple like:

The Beat module works with Beats 7.3.0 and later.

Reply:
++ to the simpler language. Thanks, will fix all of them!